cross/fish/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = fish
+PKG_FULL_NAME = fish-shell
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = git
```

Comment:
Why not use the official tarball: http://fishshell.com/files/2.1.2/fish-2.1.2.tar.gz? Probably means you can drop `PKG_FULL_NAME` and `PRE_CONFIGURE_TARGET`.